rest/configs/PaypalSandbox.oauth

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Settings>
+  <AuthorizationRequestUrl>https://www.sandbox.paypal.com/webapps/auth/protocol/openidconnect/v1/authorize?client_id={ApiKey}&amp;response_type=code&amp;scope=profile+email+address+phone+https://uri.paypal.com/services/paypalattributes&amp;redirect_uri=http://pragmaticworks.com</AuthorizationRequestUrl>
+  <AuthorizationRequestMethod>POST</AuthorizationRequestMethod>
+  <AuthorizationReturnUrl>http://localhost</AuthorizationReturnUrl>
+  <GrantCodeQueryStringParameter>code</GrantCodeQueryStringParameter>
+  <AccessTokenRequestUrl>https://api.sandbox.paypal.com/v1/identity/openidconnect/tokenservice</AccessTokenRequestUrl>
+  <AccessTokenValidationUrl></AccessTokenValidationUrl>
+  <AccessTokenRequestUrlQueryParameters>client_id={ApiKey}&amp;client_secret={ApiSecret}&amp;grant_type=authorization_code&amp;code={GrantCode}&amp;redirect_uri=http://pragmaticworks.com</AccessTokenRequestUrlQueryParameters>

```

Comment:
Same issue here

Reply:
Thank you for pointing that out, I've updated them all to the new one